### DISCUSSION TITLE
Next Release of Specification

### DISCUSSION THREAD

**Discussion by k-jay-c** [serverlessworkflow/specification#796] [General] [answered]
Any ideas when the next release is happening?

**Comment by cdavernas** [comment, marked as answer]
It's really hard to say, we still have a couple of big milestones to reach first. 

I personally hope to have a bit of time to work on some long overdue (and required) PRs this quarter, which should hurry things up.

We still need, however, to setlte on a proper error model, which I believe we should over in upcoming weeklies.

In other words, I think it's reasonable to aim for end of Q1 2024 or smtg of the like (I hope real sooner though). Wdyt @ricardozanini @tsurdilo?

Hope this helps!

**Reply by ricardozanini** [reply]
Q1 2024 is what I'm hoping for too. @k-jay-c you can track the progress here: https://github.com/orgs/serverlessworkflow/projects/1

If you want, you can take any TODO issues and engage there, hopefully with an upcoming PR.
